beacon_node/beacon_chain/src/test_utils.rs

```diff
@@ -1665,6 +1665,7 @@ where
         {
             self.chain
                 .import_attester_slashing(verified_attester_slashing);
+            // TODO add attester SSE
```

Comment:
We can remove this TODO since we register the SSE inside the `import_attester_slashing` function.